### DISCUSSION TITLE
get text vs get full text

### DISCUSSION THREAD

**Discussion by vajrang-b** [vajrang-b/RPA-Developer-in-30-Days#22] [Q&A] [answered]
what are the uses and differences

**Comment by Ulhasguttikar** [comment, marked as answer]
get text:- It uses Native Screen Scrapping. Get Text can scrap only visible data.
Get full text:- it uses Full Text screen scrapping. it can scrap data including the hidden text.

example.
if we need to scrap text out of dropdown without expanding the dropdown list
get text will scrap only visible text that is first element.
get full text will scrap all dropdown options available.

**Comment by TiwariRakesh** [comment]
Get Text- provides the visible text.
Get full text- provides the hidden text as well
